In-house patch
makes the original "00-makefile.in.patch" more general
...to be promote-able to community.

--- a/configure.ac	2016-07-20 05:53:41.592301927 -0700
+++ b/configure.ac	2016-07-20 06:31:13.486744587 -0700
@@ -10,6 +10,8 @@
 # Checks for programs.
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
+AC_PROG_LN_S
+AC_CHECK_PROG(TAR,tar,tar)
 
 SBINSEARCHPATH=/usr/sbin:/sbin:/usr/local/sbin:/etc/sbin
 BINSEARCHPATH=/usr/bin:/bin:/usr/local/bin
